src/clim2parquet/__init__.py

```diff
@@ -0,0 +1,103 @@
+# SPDX-FileCopyrightText: 2025-present Imperial College of Science and Technology
+#
+# SPDX-License-Identifier: MIT
+
+from . import tools
+
+import os
+from typing import Union
+import warnings
+
+
+def get_data_names():
```

Comment:
```suggestion
def get_data_names() -> list[str]:
```

if you're going on in on types, might as well do this entirely. I'm sure there's a configuration option somewhere that will check these for us